pkgs/development/python-modules/pyfuse3/default.nix

```diff
@@ -14,17 +14,17 @@
 
 buildPythonPackage rec {
   pname = "pyfuse3";
-  version = "3.2.3";
+  version = "3.3.0";
 
-  disabled = pythonOlder "3.5";
+  disabled = pythonOlder "3.8";
 
   format = "setuptools";
```

Comment:
Since v3.2.3, seems like this package uses pyproject.toml. Can we make that switch?

Reply:
We can, but it's really only a thin wrapper around setuptools in this case.

Reply:
Thank you for making the change. You're right that it's just a wrapper, but as I'm doing more work with setuptools, I'm starting to see that it has tons of legacy code paths that are a mess, and I hope moving as much as possible to its PEP 517 code path will be easier to maintain.